src/data/ua_overrides.js

```diff
@@ -287,7 +287,7 @@ const AVAILABLE_UA_OVERRIDES = [
     domain: "posts.google.com",
     bug: "1566253",
     config: {
-      matches: ["*://posts.google.com/share/*"],
+      matches: ["*://posts.google.com/*"],
       uaTransformer: _ => {
```

Comment:
This override worked on the original reported page, but then when clicking on "More" and trying to load more posts here https://posts.google.com/share/iT2YoKXw?sa=X&ved=0ahUKEwi7nqT01cvjAhVsg-AKHS12BQoQrpcBCAw ,
the response won't return anything :( So I've added the full domain instead.